### ISSUE TITLE
Remove duplicate entries in the REPL history

### ISSUE THREAD

**Issue by TristanCacqueray** [byorgey/swarm#75]
**Is your feature request related to a problem? Please describe.**
When repeating a command, the REPL history gets cluttered with repeated elements, rendering the history navigation tedious.

**Describe the solution you'd like**
The history should only contains unique command.

**Describe alternatives you've considered**
To avoid the confusion, the history index could be annotated to avoid confusion when the line does not change when pressing up.

**Comment by byorgey** [comment]
Agreed.  Maybe a simpler approach is to just check the previous entry when adding something to the REPL history, and don't add it if the previous entry is identical.
